hangman.js

```diff
@@ -1 +1,81 @@
-console.log("welcome to hangman");
\ No newline at end of file
+
+var hangman = {
+  word: "",
+  letter: "",
+  newWord: "",
+  welcome: alert("welcome to hangman"),
+  displayDashes: [],
+  counterAttempts: 6,
```

Comment:
A future iteration of this could include an `guessedLetters` array so that if the user guessing a wrong letter for than once, that is not registered as an incorrect guess. Instead it would say something like "You already guessed that. Try again."